.gitignore

```
@@ -2,6 +2,9 @@
 node_modules
 /dist
 
+# github token
+token.txt
```

```suggestion
github_token.txt
```

Name a file in a way that it is understandable by file name.